src/pathwayvis/components/mapselector/mapselector.component.html

```diff
@@ -2,8 +2,8 @@
   <md-card>
     <md-input-container style="margin-left: 10px;">
       <label>Model</label>
-      <md-select name="model" ng-model="ctrl.model" ng-change="ctrl.changeModel()" flex>
-        <md-option ng-repeat="model in ctrl.getModels()" ng-value="model">{{ model }}</md-option>
+      <md-select name="model" ng-model="$ctrl.model" ng-change="$ctrl.changeModel()" flex>
+        <md-option ng-repeat="model in $ctrl.getModels()" ng-value="model">{{ model }}</md-option>
```

Comment:
`$ctrl` is the default and the convention. This way there's no need to declare `controllarAs` in the component definition.